packages/yourloops/components/alarm/patient-alarm-dialog.tsx

```diff
@@ -89,27 +89,25 @@ function PatientAlarmDialog(props: PatientAlarmDialogProps): JSX.Element {
       onClose={onClose}
       data-testid="patient-alarm-dialog"
     >
-      <div className={commonClasses.root}>
-        <DialogTitle id="remote-monitoring-dialog-invite-title" className={classes.title}>
-          <div className={commonClasses.categoryHeader}>
-            <div>
-              <TuneIcon />
-              <Typography className={commonClasses.title}>
-                {t('events-configuration')}
-              </Typography>
+          <DialogTitle id="remote-monitoring-dialog-invite-title" className={classes.title}>
+            <div className={commonClasses.categoryHeader}>
+              <div>
+                <TuneIcon />
+                <Typography className={commonClasses.title}>
+                  {t('events-configuration')}
+                </Typography>
+              </div>
             </div>
-          </div>
-        </DialogTitle>
+          </DialogTitle>
 
-        <Box paddingX={3} marginBottom={2}>
-          <AlarmsContentConfiguration
-            monitoring={patient.monitoring}
-            patient={patient}
-            onSave={save}
-            saveInProgress={saveInProgress}
-            onClose={onClose} />
-        </Box>
-      </div>
+          <DialogContent className="no-padding-bottom">
```

Comment:
If we really want to go in detail, we have buttons wrapped in the DialogContent, so the space between Dialog border and buttons is too large compared to other dialogs because there is the DialogContent padding. I suggest to remove this padding for buttons and add it only around the content 😉